data/avro-schema/src/main/resources/avro/net/corda/data/membership/p2p/SignedMemberships.avsc

```diff
@@ -7,7 +7,7 @@
     {
       "name": "hashCheck",
       "doc": "Hash value to help identify out-of-sync members.",
-      "type": "net.corda.data.crypto.SecureHash"
+      "type": ["null", "net.corda.data.crypto.SecureHash"]
```

Comment:
just a question: why was this change needed?

Reply:
From the [ticket](https://r3-cev.atlassian.net/browse/CORE-2932):

>  ...start sync when no hash from the MGM is available